test/development/experimental-https-server/https-server.generated-key.test.ts

```diff
@@ -33,4 +33,13 @@ describe('experimental-https-server (generated certificate)', () => {
     const html = await renderViaHTTP(next.url, '/2', undefined, { agent })
     expect(html).toContain('Hello from Pages')
   })
+
+  it('should successfully reuse generated certificates', async () => {
```

Comment:
I would also add test case for next dev run with another hostname. In this case we should regenerate cert. 
But i`m not sure how to restart next with another cli options during test

Reply:
Maybe you can point me to a similar test